libs/clients/air-discount-scheme/README.md

```diff
@@ -2,4 +2,37 @@
 
 # Air Discount Scheme Client
 
-This library was generated with [Nx](https://nx.dev).
+This client relies on the Air Discount Scheme Backend.
+This client exposes some private methods for said backend and is intended for internal or machine client use.
+
+To generate the client run
+
+```bash
+yarn nx run clients-air-discount-scheme:schemas/build-openapi
+yarn nx run clients-air-discount-scheme:schemas/external-openapi-generator
+```
+
+## Simple usage
+
+Then in your module you can set up the imports
+
+```ts
+//your.module.ts
+import { AirDiscountSchemeClientModule } from '@island.is/clients/air-discount-scheme'
+
+@Module({
+  providers: [YourService],
+  imports: [AirDiscountSchemeClientModule],
+})
+```
+
+```ts
+//your.service.ts
+import { UsersApi as AirDiscountSchemeApi } from '@island.is/clients/air-discount-scheme'
```

Comment:
Why not just name the client `AirDiscountSchemeApi`?

Reply:
The name is generated in the openapi based on the @ApiTag('Users') decorator